client/src/components/Tool/ToolHelp.vue

```diff
@@ -1,30 +1,14 @@
 <template>
-    <div class="form-help form-text mt-4" v-html="formattedContent" />
+    <div class="form-help form-text mt-4" v-html="content" />
 </template>
 
 <script>
-import $ from "jquery";
-import { getAppRoot } from "onload/loadConfig";
-
 export default {
     props: {
         content: {
             type: String,
             required: true,
         },
     },
-    computed: {
-        formattedContent() {
-            const $tmpl = $("<div/>").append(this.content);
```

Comment:
It looks like this substantively changes the incoming content, to me, and that functionality would need to be updated?

i.e. -- it makes all of the anchors target a new window, and it rewrites static references in the tool help.